### PR TITLE
Bumping version and updating changelog for 2.2.0-beta1

### PR DESCRIPTION
**Pull request by mikedickey**
Also fixing a compiler warning with Settings.h when NO_GUI

### CODE REVIEW

**Review by nwang92 — APPROVED** [review]

**Comment by mikedickey** [comment]
@ntonnaett any idea why the flatpak build is failing for beta releases?

**Comment by ntonnaett** [comment]
> @ntonnaett any idea why the flatpak build is failing for beta releases?

It seems appstreamcli fails when using xvfb-run. I'm not sure why it tries to run an x11 application. There is no change from our side I could find, that could be responsible for that. Maybe the github actions container changed or the github action module we use.

**Comment by mikedickey** [comment]
> > @ntonnaett any idea why the flatpak build is failing for beta releases?
> 
> It seems appstreamcli fails when using xvfb-run. I'm not sure why it tries to run an x11 application. There is no change from our side I could find, that could be responsible for that. Maybe the github actions container changed or the github action module we use.

What is especially odd is that it's running fine right now on dev. Only if I change the version string in jacktrip_globals.h does it break.

**Comment by ntonnaett** [comment]
#1238 updates the flatpak-builder module and builds fine. If we merge this and it still doesn't work with this one, it's probably not the action itself. Maybe different behavior because of a different trigger.

File: docs/changelog.yml [hunk 1]
```diff
@@ -1,8 +1,11 @@
-- Version: "2.1.1"
-  Date: 2023-12-xx
+- Version: "2.2.0"
+  Date: 2023-01-16
```
Comment (ntonnaett) [inline docs/changelog.yml:2]: 2024

**Comment by mikedickey** [comment]
@ntonnaett Thanks, you called it! Turned out to be the ampersand in the changelog.yml.

**Pull request merged**